tox.ini

```diff
@@ -7,15 +7,16 @@ envlist =
     # the version matrix according to the Django docs:
     # https://docs.djangoproject.com/en/2.0/faq/install/#what-python-version-can-i-use-with-django
     py{27,36}-dj111-tests,
-    py{36,37}-dj{21,22}-tests,
+    py{36,37,38}-dj{21,22}-tests,
     py{27,36}-fl{011,012,10}-tests,
-    py{36,37}-s19-tests,
+    py{36,37,38}-s19-tests,
```

Comment:
Python `3.8` is not in `.travis.yml` so tests aren't run on that version in travis: https://travis-ci.org/mozilla-services/python-dockerflow/builds/602217901